src/index.js

```diff
@@ -13,7 +14,18 @@ ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
       <header>
-        <h1>Rotten<br/> Tomatillos</h1>
+        <h1>
+          <Link to={{
+            pathname:'/'
+          }}  className="site-title">Rotten<br/> Tomatillos
+          </Link>
+        </h1>
+        <Link to={{
+          pathname:'/'
+        }}>
+          <img src={homeButton} className='home-button'/>
+        </Link>
```

Comment:
Love the touch of the title also taking you back to the home page! I personally use that when I visit sites and feel not as many people know about this!

Reply:
Yes, excellent UX!